Doc/library/ftplib.rst

```diff
@@ -102,12 +102,15 @@ FTP objects
 
    .. method:: FTP.set_debuglevel(level)
 
-      Set the instance's debugging level.  This controls the amount of debugging
-      output printed.  The default, ``0``, produces no debugging output.  A value of
-      ``1`` produces a moderate amount of debugging output, generally a single line
-      per request.  A value of ``2`` or higher produces the maximum amount of
-      debugging output, logging each line sent and received on the control connection.
-
+      Set the instance's debugging level as an :class:`int`.
+      This controls the amount of debugging output printed.
+      The debug levels are:
+
+      * ``0`` (the default): No debug output.
```

Comment:
?
```suggestion
      * ``0`` (default): No debug output.
```

Reply:
I have no strong opinion about that. I see that both variants are used in the Docs with 57 occurrences of `(default)` and 140 of `(the default)`.